P/Pango/build_tarballs.jl

```diff
@@ -3,24 +3,48 @@
 using BinaryBuilder
 
 name = "Pango"
-version = v"1.50.9"
+version = v"1.50.14"
 
 # Collection of sources required to build Pango
 sources = [
     ArchiveSource("http://ftp.gnome.org/pub/GNOME/sources/pango/$(version.major).$(version.minor)/pango-$(version).tar.xz",
-                  "1b636aabf905130d806372136f5e137b6a27f26d47defd9240bf444f6a4fe610"),
+                  "1d67f205bfc318c27a29cfdfb6828568df566795df0cb51d2189cde7f2d581e8"),
+    DirectorySource("./bundled"),
+    ArchiveSource("https://sourceforge.net/projects/mingw-w64/files/mingw-w64/mingw-w64-release/mingw-w64-v11.0.0.tar.bz2",
+                  "bd0ea1633bd830204cc23a696889335e9d4a32b8619439ee17f22188695fcc5f"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
+
+if [[ "${target}" == *-mingw* ]]; then
+    cd $WORKSPACE/srcdir/mingw*/mingw-w64-headers
+    ./configure --prefix=/opt/$target/$target/sys-root --enable-sdk=all --host=$target
+    make install
+
+    cd ../mingw-w64-crt/
+    if [ ${target} == "i686-w64-mingw32" ]; then
+        _crt_configure_args="--disable-lib64 --enable-lib32"
+    elif [ ${target} == "x86_64-w64-mingw32" ]; then
+        _crt_configure_args="--disable-lib32 --enable-lib64"
+    fi
+    ./configure --prefix=/opt/$target/$target/sys-root --enable-sdk=all --host=$target --enable-wildcard ${_crt_configure_args}
+    make -j${nproc}
+    make install
+fi
```

Comment:
Why do you need this?

Reply:
Without it, `x86_64-w64-mingw32`:
```
[07:00:12] ../pango/pangowin32-dwrite-fontmap.cpp: In function ‘PangoWin32DWriteItems* pango_win32_init_direct_write()’:
[07:00:12] ../pango/pangowin32-dwrite-fontmap.cpp:37:33: error: ‘IID_IDWriteFactory’ was not declared in this scope
[07:00:12]  # define UUID_OF_IDWriteFactory IID_IDWriteFactory
```

I copied this from the GTK4 recipe, which came from the Qt6 recipe: https://github.com/JuliaPackaging/Yggdrasil/pull/6590